ansible_collections/arista/avd/roles/eos_designs/schemas/eos_designs.schema.yml

```diff
@@ -4605,6 +4605,12 @@ $defs:
               '
             type: str
             format: ipv4_cidr
+          mlag_compact_addressing:
+            description: |
+              Generate MLAG ip addresses based on the odd ID of an ID pair, this a /31 per allocation
+              Requires MLAG pairs have nodes with an odd and even ID
+            default: false
+            type: bool
```

Comment:
I would prefer to build something more generic, since we will have other algorithms going forward (like reusing the same subnet everywhere).
```yaml
ip_addressing:
  mlag:
    algorithm: < 'first_id' | 'odd_id' | default -> 'first_id' >
```
Make sure to describe the two in the description for `algorithm` including the downside of skipping every other subnet for first_id (in favor of flexibility in IDs) and the more restrictive ID scheme for 'odd_id'

Reply:
Added